Formula/t/tdb.rb

```diff
@@ -0,0 +1,27 @@
+class Tdb < Formula
+  desc "Trivial DataBase, by the Samba project"
+  homepage "https://tdb.samba.org/"
+  url "https://www.samba.org/ftp/tdb/tdb-1.4.10.tar.gz"
+  sha256 "02338e33c16c21c9e29571cef523e76b2b708636254f6f30c6cf195d48c62daf"
+  license "GPL-3.0-or-later"
+
+  uses_from_macos "python" => :build
+  on_linux do
+    depends_on "python@3.12" => :build
+  end
```

Comment:
this would be redundant?

Reply:
Tdb needs to be able to run python to build. The formula wasn't working until I added it as a dependency. On macOS I can rely on the system one, but it seems that at least on the GitHub runners it needs to be explicitly specified

Reply:
makes sense but `uses_from_macos` is sufficient to also install python on linux (since it is functionally equivalent to this `on_linux` block), so we can remove it

Reply:
Removed, confirmed working. Thanks!